yarn.lock

```diff
@@ -5,7 +5,6 @@
 "@rails/webpacker@3.5":
   version "3.5.5"
   resolved "https://registry.yarnpkg.com/@rails/webpacker/-/webpacker-3.5.5.tgz#8911c66bcefc8bc6b91270e92f0d39e3c2d43116"
-  integrity sha512-R9kglLb/R9mMqf2BW91XJHYc0q00qAO67vfrbV52ifTSywPOixEka/7oU3cb6671u1ajd74kuGAsCSnBtdVHJg==
```

Comment:
This PR removes all the integrity sha's from yarn.lock

Reply:
I did a fresh `yarn install` after deleting `yarn.lock` and I didn't see this integrity `sha`

Reply:
@yachtcaptain23 is your yarn up-to-date? I think we probably want them.  cc @evq who added to the codebase originally https://github.com/brave-intl/publishers/pull/1189#issuecomment-424879428